README.md

```diff
@@ -19,6 +19,8 @@
   - [React](#react)
   - [Ruby / Rails](#ruby--rails)
   - [Other Non-Technical Topics](#other-non-technical-topics)
+  - [Angular](#angular)
```

Comment:
Angular is already there and please put Python before react :)

Reply:
oh you are right
Python before react just here or also in the soapbox list at the bottom ?

Reply:
Ah, probably both, good catch!

Reply:
@theothermattm done!
I also added a "System dependencies" section